src/Core/Prefab/GlobeView.js

```diff
@@ -371,10 +371,17 @@ GlobeView.prototype.readDepthBuffer = function readDepthBuffer(x, y, width, heig
 
 const matrix = new THREE.Matrix4();
 const screen = new THREE.Vector2();
-const pickWorldPosition = new THREE.Vector3();
 const ray = new THREE.Ray();
 const direction = new THREE.Vector3();
-GlobeView.prototype.getPickingPositionFromDepth = function getPickingPositionFromDepth(mouse) {
+
+/**
+ * returns the pick position under the mouse
+ *
+ * @param      {THREE.Vector2}  mouse   mouse coordinates to pick position
+ * @param      {THREE.Vector3}  target optional Vector3 target. the result will be copied into this Vector3. If not present a new one will be created
+ * @return     {THREE.Vector3}  The picking position
```

Comment:
Obviously it will return "The picking position", because it's already written in the name of the function.

The documentation must specify what's missing from the function name (eg: units)